android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -40,7 +40,8 @@
     private FeedAdapter feedAdapter;
     private List<FeedItem> feedItems;
     private ProgressDialog downloadProgressDialog;
-    private Disposable feedLoaderDisposable;
+    private Disposable feedDownloaderDisposable;
+    private Disposable refreshFeedDisposable;
```

Comment:
Use a final CompositeDisposable instead. This makes us remove the null checks as well